### PR TITLE
Add iframe to code example box

### PR DESCRIPTION
**Pull request by gi4no**
Closes https://github.com/themesberg/flowbite-svelte/issues/1062
## 📑 Description

Add iframe in the code example for handle breakpoint correctly, the iframe is render only when responsive buttons are enabled

## Status

- [ ] Not Completed
- [x] Completed

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] I have checked the page with https://validator.unl.edu/
- [x] All the tests have passed
- [x] My pull request is based on the latest commit (not the npm version).

### CODE REVIEW

File: src/routes/utils/ExampleWrapper.svelte [hunk 1]
```diff
@@ -8,6 +8,7 @@
   import type { PageData } from '../$types';
   import { identity } from 'svelte/internal';
```
Comment (gi4no) [inline src/routes/utils/ExampleWrapper.svelte]: @shinokada this import is not used, is it necessary?

**Comment by gi4no** [comment]
@shinokada 
I wanted to provide some insights on this pull request:

It's been quite a journey, to say the least, and it was a bit of a challenge. Initially, the solution worked well on Chrome but had issues with Firefox due to iframe loading.

The current solution, which relies on the iframe's onload event, works well across browsers, but there's a minor hiccup with Chrome ( https://github.com/sveltejs/svelte/issues/6967 ). To address this, I've added a workaround.

Your feedback on this would be greatly appreciated.

There is only one bug that I see, on first load the iframe doesn't load the css, I need to look into the problem further

I've to perform some other test and if everything is fine i mark the RP completed.

thank you

File: src/routes/utils/ExampleWrapper.svelte (hunk 1 above)
Comment (shinokada) [reply src/routes/utils/ExampleWrapper.svelte]: You can remove `identity`, but `PageData` is used in line 26.

**Comment by shinokada** [comment]
Some of the examples need higher heights:

- all examples in https://flowbite-svelte-m512xpqvw-themesberg.vercel.app/docs/components/navbar
- Some examples in the card page

As you mentioned, it won't work without refreshing/reloading the browser.

**Comment by gi4no** [comment]
@shinokada 
I've resolved the CSS problem and fix the NavBar page.

I don't understand what is wrong with the card example. Can you please be more precise?

**Comment by shinokada** [comment]
I thought a bit more space for the top and bottom of the card examples will be nicer.

image

**Comment by shinokada** [comment]
While working on the example box, can you have a look at this issue, https://github.com/themesberg/flowbite-svelte/issues/1062?

**Comment by gi4no** [comment]
> While working on the example box, can you have a look at this issue, #1062?

I found a solution, simply tidying up the preprocess in `svelte.config.js`

```ts
...
preprocess: [
    mdsvex(mdsvexConfig), <-- this before the svelte preprocess
    preprocess({
      postcss: true
    })
  ],
...
```

Screenshot 2023-09-14 at 01 54 53

do I add the change in `svelte.config.js` in this PR?

**Comment by shinokada** [comment]
Yes, please. Thank you.

**Comment by gi4no** [comment]
@shinokada I'm done, tell me what you think 😄

**Comment by shinokada** [comment]
Thank you for your work.

1. I noticed that [Navbar with dropdown](https://flowbite-svelte-7rqwa8mhm-themesberg.vercel.app/docs/components/navbar#Navbar_with_dropdown) is not working.

2. Can you add a bigger height value for [all the mega-menus mobile views](https://flowbite-svelte-7rqwa8mhm-themesberg.vercel.app/docs/components/mega-menu#Default_mega_menu)?

**Comment by shinokada** [comment]
Thank you for your PR! Your contributions are greatly appreciated, and I'm looking forwards to more of your future PRs.

**Pull request merged**